src/main/java/com/zontik/groshiky/service/TransactionService.java

```diff
@@ -48,26 +49,25 @@ public void deleteTransactionById(Integer id) {
 
     @Override
     public Transaction findTransactionById(Integer id) {
-        return transactionRepository.getOne(id);
+        return transactionRepository.findById(id).orElse(null);
```

Comment:
better .orElseThrow(...) because if frontend asking for an entity that doesn't exists - it's exceptional case